.docs/Docker.md

```diff
@@ -45,7 +45,7 @@ Note the `-v /path/on/machine:/out` option, which instructs Docker to bind the `
 > **Note**:
 > If you are running SELinux, you will need to add the `:z` option after `/out`, e.g.:
 >
-> ```
+> ```powershell
```

Comment:
Maybe we should use `console` for these too?

Reply:
Yup, I will change those.